### PR TITLE
redirecting http:// to https://

### PR DESCRIPTION
**Pull request by vamsi963601**
Added an Express Middleware for HTTPS redirection.

### CODE REVIEW

File: web/server.js [hunk 1]
```diff
@@ -32,6 +32,8 @@ const STATIC_PATH = process.env.STATIC_PATH || (`${__dirname}/public`);
 app.use(compression()); // Cloudflare already does gzip but we do it anyway
 app.disable('etag'); // Disable etag generation (except for static)
 app.use((req, res, next) => {
+  if (req.headers['x-forwarded-proto'] !== 'https')
+      return res.redirect('https://' + req.headers.host + req.url);
```
Comment (skovhus) [inline web/server.js:36]:
```suggestion
  if (!req.secure) {
      return res.redirect('https://' + req.headers.host + req.url);
  }
```

**Review by skovhus — COMMENTED** [review]
Thanks for the contribution! I've added a suggestion to use the `req.secure` (see http://expressjs.com/en/api.html) for readability.

**Comment by vamsi963601** [comment]
Thanks for letting me know the updated methods.

**Review by skovhus — APPROVED** [review]
Thanks for fixing this and welcome as an electricityMapper! 🙏

**Pull request merged**